beacon-chain/slasher/detect_attestations.go

```diff
@@ -0,0 +1,62 @@
+package slasher
+
+import (
+	"context"
+
+	types "github.com/prysmaticlabs/eth2-types"
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+)
+
+// Process queued attestations every time an epoch ticker fires. We retrieve
+// these attestations from a queue, then group them all by validator chunk index.
+// This grouping will allow us to perform detection on batches of attestations
+// per validator chunk index which can be done concurrently.
+func (s *Service) processQueuedAttestations(ctx context.Context, epochTicker <-chan uint64) {
+	for {
+		select {
+		case currentEpoch := <-epochTicker:
+			atts := s.attestationQueue
+			s.attestationQueue = make([]*ethpb.IndexedAttestation, 0)
+			log.Infof("Epoch %d reached, processing %d queued atts for slashing detection", currentEpoch, len(atts))
```

Comment:
This maybe too verbose for info. Consider debug

Reply:
We do want to give a user more feedback if they are running a slasher beacon node, given this log is only triggered per epoch it shouldn't be too verbose IMO

Reply:
Sound good. Then consider using `WithFields`. Easier to parse down the line